src/libraries/System.Net.Http/src/System/Net/Http/ByteArrayHelpers.cs

```diff
@@ -2,61 +2,26 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System.Diagnostics;
+using System.Text;
 
 namespace System
 {
     internal static class ByteArrayHelpers
```

Comment:
Can you please delete this helper class entirely? There are only ~5 places that call into this which can now use the Ascii API directly.

Reply:
My goal was to add the debug asserts to verify that at least one of the inputs is always ASCII. But the CI is green, so it's confirmed and I can remove it now.

Reply:
These APIs may be used where neither input is fully ascii and they need to account for that, regardless, of course. I believe we agreed they could simply return false in such cases (though wouldn't be guaranteed to).